src/ServiceControl.Config/UI/AdvancedOptions/ServiceControlAdvancedViewModel.cs

```diff
@@ -17,7 +17,7 @@
 
     class ServiceControlAdvancedViewModel : RxProgressScreen, IHandle<RefreshInstances>
     {
-        public ServiceControlAdvancedViewModel(BaseService instance, IEventAggregator eventAggregator, StartServiceControlInMaintenanceModeCommand maintenanceModeCommand, DeleteServiceControlInstanceCommand deleteInstanceCommand)
+        public ServiceControlAdvancedViewModel(BaseService instance, IEventAggregator eventAggregator, StartServiceControlInMaintenanceModeCommand maintenanceModeCommand, DeleteServiceControlInstanceCommand deleteInstanceCommand, ForceUpgradeServiceControlInstanceCommand forceUpgradeCommand)
```

Comment:
With so many dependencies easier to understand when listed on separate lines:
```suggestion
        public ServiceControlAdvancedViewModel(
            BaseService instance,
            IEventAggregator eventAggregator,
            StartServiceControlInMaintenanceModeCommand maintenanceModeCommand,
            DeleteServiceControlInstanceCommand deleteInstanceCommand,
            ForceUpgradeServiceControlInstanceCommand forceUpgradeCommand
            )
```